apps/docs-snippets/src/guide/contracts/contract-balance.test.ts

```diff
@@ -15,8 +15,7 @@ describe(__filename, () => {
 
   it('should successfully get a contract balance', async () => {
     // #region contract-balance-3
-    // #context import type { AssetId } from 'fuels';
-    // #context import { Wallet, BN, BaseAssetId } from 'fuels';
+    // #import { AssetId, Wallet, BN, BaseAssetId }
```

Comment:
Just reinforcing what I said already, unless there's a reason behind it, I suggest keeping these imports with an ending semi-colon to keep the standards and avoid variations and unnecessary questions.

```
// #import { AssetId, Wallet, BN, BaseAssetId };
```